assets/training/finetune_acft_image/environments/acpt_image_huggingface/context/requirements.txt

```diff
@@ -0,0 +1,8 @@
+datasets==2.12.0
+requests==2.31.0 # Vulnerability fix
```

Comment:
do not pin requests, the latest will be installed

Reply:
Done